spellings/cspell.config.yaml

```diff
@@ -0,0 +1,22 @@
+---
+$schema: https://raw.githubusercontent.com/streetsidesoftware/cspell/main/cspell.schema.json
+version: '0.2'
+# Allows things like stringLength
+allowCompoundWords: true
+useGitignore: true
+# Could split this up? And do a dictionary for each repo?
+# But feel like if this isn't super slow
+# That having just one single dictionary might be nicer?
+dictionaryDefinitions:
+  - name: freertos-words
+    path: '.cSpellWords.txt'
```

Comment:
Should this file be present in this PR? How do I know the dictionary of words hasn't changed?

Reply:
Idea is you would change the dictionary of words if you add in new words, it's the same thing that happens currently with needing to add new words to the lexicon.txt file